code/game/world.dm

```diff
@@ -7,6 +7,78 @@
 
 GLOBAL_VAR(restart_counter)
 
+/**
+ * WORLD INITIALIZATION
+ * THIS IS THE INIT ORDER:
+ *
+ * BYOND =>
+ * - (secret init native) =>
+ *   - world.RealGlobalsPreInit() =>
+ *     - world.init_byond_tracy()
+ *     - (Start native profiling)
+ *     - (Start debugger if present)
+ *     - Master =>
+ *       - config *unloaded
+ *       - (all subsystems) PreInit()
+ *       - GLOB =>
+ *         - make_datum_reference_lists()
+ * - (all pre-mapped atoms) /atom/New()
+ * - world.New() =>
+ *   - config.Load()
+ *   - world.InitTgs() =>
+ *     - TgsNew() *may sleep
+ *     - GLOB.rev_data.load_tgs_info()
+ *   - world.ConfigLoaded() =>
+ *     - SSdbcore.InitializeRound()
+ *     - world.SetupLogs()
+ *     - load_admins()
+ *     - ...
+ *   - Master.Initialize() =>
+ *     - (all subsystems) Initialize()
+ *     - Master.StartProcessing() =>
+ *       - Master.Loop() =>
+ *         - Failsafe
```

Comment:
RFC @MrStonedOne for two things:
- `load_admins()` used to be before `InitializeRound()` (previously `SetRoundID()`) but I've moved it so any errors that occur during the admin load go to the `sql.log` instead of `config_error.log`.
- Has the Failsafe always loaded so late??